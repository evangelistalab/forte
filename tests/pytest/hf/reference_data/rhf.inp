molecule {
    H 0.0 0.0 0.0
    H 0.0 0.0 1.0
}

set scf_type pk
set basis cc-pVDZ

energy('scf')
