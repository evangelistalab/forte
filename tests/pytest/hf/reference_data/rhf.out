
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 undefined 

                         Git: Rev {master} 966d1bd 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, and M. H. Lechner

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Saturday, 29 May 2021 11:22AM

    Process ID: 58537
    Host:       CHE-FEVANGE
    PSIDATADIR: /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
    H 0.0 0.0 0.0
    H 0.0 0.0 1.0
}

set scf_type pk
set basis cc-pVDZ

energy('scf')
--------------------------------------------------------------------------

Scratch directory: /tmp/

*** tstart() called on CHE-FEVANGE
*** at Sat May 29 11:22:23 2021

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry H          line    22 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D_inf_h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         H            0.000000000000     0.000000000000    -0.500000000000     1.007825032230
         H            0.000000000000     0.000000000000     0.500000000000     1.007825032230

  Running in d2h symmetry.

  Rotational constants: A = ************  B =     33.45348  C =     33.45348 [cm^-1]
  Rotational constants: A = ************  B = 1002910.21295  C = 1002910.21295 [MHz]
  Nuclear repulsion =    0.529177210670000

  Charge       = 0
  Multiplicity = 1
  Electrons    = 2
  Nalpha       = 1
  Nbeta        = 1

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 6
    Number of basis functions: 10
    Number of Cartesian functions: 10
    Spherical Harmonics?: true
    Max angular momentum: 1

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:               6
      Number of primitives:             10
      Number of atomic orbitals:        10
      Number of basis functions:        10

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 3080 doubles for integral storage.
  We computed 231 shell quartets total.
  Whereas there are 231 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 2.3377152309E-01.
  Reciprocal condition number of the overlap matrix is 1.2474073930E-01.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     Ag         3       3 
     B1g        0       0 
     B2g        1       1 
     B3g        1       1 
     Au         0       0 
     B1u        3       3 
     B2u        1       1 
     B3u        1       1 
   -------------------------
    Total      10      10
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:    -0.83821031082801   -8.38210e-01   0.00000e+00 
   @RHF iter   1:    -1.09846572476544   -2.60255e-01   8.25729e-03 DIIS
   @RHF iter   2:    -1.10010178613905   -1.63606e-03   1.43770e-03 DIIS
   @RHF iter   3:    -1.10015371698349   -5.19308e-05   5.94752e-05 DIIS
   @RHF iter   4:    -1.10015376478315   -4.77997e-08   6.42536e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag    -0.525044  

    Virtual:                                                              

       1B1u    0.143588     2Ag     0.539607     2B1u    0.794799  
       1B2u    1.287130     1B3u    1.287130     3Ag     1.679162  
       1B2g    1.871551     1B3g    1.871551     3B1u    2.735411  

    Final Occupation by Irrep:
             Ag   B1g   B2g   B3g    Au   B1u   B2u   B3u 
    DOCC [     1,    0,    0,    0,    0,    0,    0,    0 ]

  @RHF Final Energy:    -1.10015376478315

   => Energetics <=

    Nuclear Repulsion Energy =              0.5291772106699999
    One-Electron Energy =                  -2.2085759215581202
    Two-Electron Energy =                   0.5792449461049715
    Total Energy =                         -1.1001537647831490

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.0000

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.0000

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.0000     Total:     0.0000

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:     0.0000     Total:     0.0000


*** tstop() called on CHE-FEVANGE at Sat May 29 11:22:25 2021
Module time:
	user time   =       0.37 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       0.37 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes

    Psi4 stopped on: Saturday, 29 May 2021 11:22AM
    Psi4 wall time for execution: 0:00:02.04

*** Psi4 exiting successfully. Buy a developer a beer!
